web/js/waste.js

```diff
@@ -1,5 +1,10 @@
 $(function() {
     $('form.waste input[type="submit"]').prop('disabled', false);
+    window.addEventListener("pageshow", function() {
```

Comment:
If this were moved to line 1 (outside the $( ready setup, as we're just adding a listener), then presumably line 2 wouldn't be needed, because the event would then fire on the page load that is initially running this JS? Worth a test anyway, so it's only needed once if so.

And then... if we don't have a pageshow, but a pagehide that does this - would that be better? That way we know we've "left" the page in an okay state, so we don't need to do anything when we're coming back to it (however we are).

Reply:
Yes, moving this to line 1 still works. And using `pagehide` is a good idea too 👍